### DISCUSSION TITLE
Need help with displaying splash screen (Windows and MacOS)

### DISCUSSION THREAD

**Discussion by GeoArkadeep** [beeware/toga#2508] [Debugging help]
Hi, I have used toga to make the GUI for my open source geomechanics software. In the documentation of the pyproject.toml file, there are mentions of a splash screen display of an image (but no details on what format goes with what OS, as defined explicitly for the program icon images). I have tried using pngs, but didn't get any splash screen either on Windows nor on MacOS. Can someone point me at the right direction? Is this at all possible with toga?

**Comment by freakboy3742** [comment]
This isn't a Toga question - Splash screens are part of app packaging, not the GUI toolkit.

*Briefcase* will include a splash screen on an app if that is something the platform supports - but desktop platforms don't support splash screens. This is described in the documentation for [macOS](https://briefcase.readthedocs.io/en/stable/reference/platforms/macOS/app.html#splash-image-format) and [Windows](https://briefcase.readthedocs.io/en/stable/reference/platforms/windows/app.html#splash-image-format).

**Reply by GeoArkadeep** [reply]
Thanks for the super fast response 😊. And I apologise for confusing a briefcase issue with a toga issue. It's isn't a deal breaker and I love toga. Thanks again 😊
